### PR TITLE
Add group prefix + configure for dev

### PR DESCRIPTION
**Pull request by illusional**

### CODE REVIEW

File: cpg_infra/config.py [hunk 1]
```diff
@@ -166,6 +166,7 @@ class GCP(DeserializableDataclass):
     # delete, which can end up with missing permissions. To implement the first phase
     # (delete), simply change this to 'True', then revert to reapply group memberships
     disable_group_memberships: bool = False
+    group_prefix: str = ''
```
Comment (lgruen) [inline cpg_infra/config.py:169]: Could you add a comment here describing when it's useful to set this?
(Could maybe default this to `None` too?)

**Review by lgruen — APPROVED** [review]
Very helpful, thanks!

File: stack/dev.yaml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 cpginfradev:
+  autoclass: true
```
Comment (lgruen) [inline stack/dev.yaml:2]: Accidentally included in this PR?

Comment (illusional) [reply stack/dev.yaml:2]: Addressing both issues affecting dev stack ;)

File: cpg_infra/config.py (hunk 1 above)
Comment (illusional) [reply cpg_infra/config.py]: Resolved :)

**Pull request merged**